app/views/accompaniment_leader/activities/_activity.html.erb

```diff
@@ -44,12 +44,15 @@
     <% else %>
       <%= link_to 'Add Accompaniment Leader Notes', new_community_accompaniment_leader_activity_accompaniment_report_path(current_community.slug, activity) %>
     <% end %>
+  <% if activity.accompaniable? %>
+    <div><em><%= activity.volunteers_needed%> </em></div>
+  <% end %>
   </div>
 
   <div class='confirm'>
     <% if current_user.attending?(activity) %>
       <button type='button' class='btn btn-success' data-toggle='modal' data-target="#activity_<%= activity.id %>_accompaniment_modal">Edit RSVP</button>
-    <% else %>
+    <% elsif activity.accompaniable? %>
```

Comment:
Because accompaniment leaders should not be factored into the count, this conditional can be removed.  But it looks like this conditional is missing in app/views/activities/_activity.html.erb